### PR TITLE
Remove unifiedDashboard feature flag.

### PR DESCRIPTION
**Pull request by asvinb**
## Summary

Addresses issue:

- #5047 

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [ ] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/modules/adsense/datastore/base.js [hunk 1]
```diff
@@ -39,9 +38,6 @@ const baseModuleStore = Modules.createModuleStore( 'adsense', {
 		'webStoriesAdUnit',
 		'autoAdsDisabled',
 	],
-	adminPage: isFeatureEnabled( 'unifiedDashboard' )
```
Comment (eugene-manuilov) [inline assets/js/modules/adsense/datastore/base.js:42]: So, we don't use this `adminPage` property anymore in our modules, therefore we can remove it from `createInfoStore` and `createModuleStore` factories respectfully and use just `googlesitekit-dashboard` in the `getAdminScreenURL` selector.

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Awesome work, @asvinb! All looks good to me. Just added one suggestion to remove a little bit more 😄

File: assets/js/modules/adsense/datastore/base.js (hunk 1 above)
Comment (asvinb) [reply assets/js/modules/adsense/datastore/base.js:42]: Sounds good @eugene-manuilov . Updated!

**Review by eugene-manuilov — APPROVED** [review]
LGTM. Thanks, @asvinb.

**Pull request merged**